src/components/ux-radio/ux-radio.hbs

```diff
@@ -0,0 +1,12 @@
+
+    <label>{{.label}}</label>
+    {{#each items}}
+        <input {{#if .type}}type="{{.type}}"{{/if}}
+            {{#if .name}}name="{{.name}}"{{/if}}
+            {{#if .id}}id="{{.id}}"{{/if}}
+            {{#if .value}}value="{{.value}}"{{/if}}
+               on-blur="handleBlur"
+               on-tap="handleTap">
```

Comment:
Also use semantic names for events. Name an event by its "intent" rather
than concrete action.

See ractive docs for details on event naming.
On 2 Nov 2015 9:53 a.m., "Nathan Flew" <notifications@github.com> wrote:

> In src/components/ux-radio/ux-radio.hbs
> <https://github.com/ractive-foundation/ractive-foundation/pull/277#discussion_r43589159>
> :
>
> > @@ -0,0 +1,12 @@
> > +
> > +    <label>{{.label}}</label>
> > +    {{#each items}}
> > +        <input {{#if .type}}type="{{.type}}"{{/if}}
> > +            {{#if .name}}name="{{.name}}"{{/if}}
> > +            {{#if .id}}id="{{.id}}"{{/if}}
> > +            {{#if .value}}value="{{.value}}"{{/if}}
> > +               on-blur="handleBlur"
> > +               on-tap="handleTap">
>
> If these are validation purposes you can merge them like
> on-blur-tap="validate" or something similar.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ractive-foundation/ractive-foundation/pull/277/files#r43589159>
> .
>